src/hooks/useCurrentRoute.ts

```diff
@@ -0,0 +1,32 @@
+import { useRouter } from 'next/router'
+import { texts } from 'src/i18n'
+
+export type Path = '/actions' | '/pois' | '/messages' | '/'
+
+const routeTitles: Record<Path, string> = {
+  '/actions': texts.nav.pageTitles.actions,
+  '/pois': texts.nav.pageTitles.pois,
+  '/messages': texts.nav.pageTitles.messages,
+  '/': '',
+}
+
+export function useCurrentRoute() {
+  const router = useRouter()
+  let currentRoute: Path
+  let routeTitle
+  if (router.pathname.includes('/actions')) {
+    currentRoute = '/actions'
+    routeTitle = routeTitles[currentRoute]
+  } else if (router.pathname.includes('/pois')) {
+    currentRoute = '/pois'
+    routeTitle = routeTitles[currentRoute]
+  } else if (router.pathname.includes('/messages')) {
+    currentRoute = '/messages'
+    routeTitle = routeTitles[currentRoute]
+  } else {
+    currentRoute = '/'
+    routeTitle = routeTitles[currentRoute]
+  }
+
+  return { currentRoute, routeTitle }
+}
```

Comment:
Toujours pas de let ;) 

```suggestion
interface CurrentRoute {
  currentRoute: Path;
  routeTitle: string;
}

export function useCurrentRoute(): CurrentRoute {
  const router = useRouter()

  if (router.pathname.includes('/actions')) {
    return {
      currentRoute: '/actions',
      routeTitle: routeTitles[currentRoute]  
    }
  }
  
  if (router.pathname.includes('/pois')) {
    return {
      currentRoute: '/pois',
      routeTitle: routeTitles[currentRoute],
    }
  }
  
  if (router.pathname.includes('/messages')) {
    return {
      currentRoute: '/messages',
      routeTitle: routeTitles[currentRoute],
    }
  }
  
  return {
    currentRoute: '/'
    routeTitle: routeTitles[currentRoute]
  }
}
```